### PR TITLE
update utlities.js to utilize AMD loading

### PR DESCRIPTION
**Pull request by tonyurquidi**
updated to utilize AMD loading. I had to return the function qs and update main.js which utilized it. Other functions in the module didn't seem to be used outside of the module so no need to return?

### CODE REVIEW

File: src/app/js/utilities.js [hunk 1]
```diff
@@ -1,312 +1,327 @@
...
+define([
+    "esri/Graphic"
+],function(Graphic){
+//This file should include miscellaneous repeatable functions used in multiple places in the code. 
+
+    Number.prototype.MagFormat = function () {
+        return this.toFixed(1);
+    };
+
+    var utils = {
```
Comment (kellybigley) [inline src/app/js/utilities.js]: This is definitely the way to go here.  We will eventually want to add all the functions in this file into this utils.

File: src/app/js/utilities.js [hunk 2]
```diff
@@ -1,312 +1,327 @@
...
-            representativeCache[id] = data;
-            resolve(data);
-        }).fail(function (err) {
-            reject(err);
+
+    // function qs(key) {
+    //     key = key.replace(/[*+?^$.\[\]{}()|\\\/]/g, "\\$&"); // escape RegEx meta chars
+    //     var match = location.search.match(new RegExp("[?&]" + key + "=([^&]+)(&|$)"));
+    //     return match && decodeURIComponent(match[1].replace(/\+/g, " "));
+    // }
```
Comment (kellybigley) [inline src/app/js/utilities.js]: lets go ahead and delete this

File: src/app/js/utilities.js [hunk 3]
```diff
@@ -1,312 +1,327 @@
...
         }
+        return returnLabel;
+    };
+
+    app.showInThousands = function (value) {
+        console.log(value);
+    };
+
+    app.AddHighlightGraphics = function (features, zoomTo) {
+        // require(["esri/Graphic"], function (Graphic) {
```
Comment (kellybigley) [inline src/app/js/utilities.js]: go head and delete this

File: src/app/js/utilities.js [hunk 4]
```diff
@@ -1,312 +1,327 @@
...
+                });
+                gfx.push(g);
+            }
+            let gfxLayer = app.map.findLayerById("gfxLayer");
+            gfxLayer.addMany(gfx);
+
+            if (zoomTo) {
+                app.view.goTo(gfx);
+            }
+        // });
```
Comment (kellybigley) [inline src/app/js/utilities.js]: delete

Comment (tonyurquidi) [reply src/app/js/utilities.js]: removed

File: src/app/js/utilities.js (hunk 3 above)
Comment (tonyurquidi) [reply src/app/js/utilities.js]: removed

File: src/app/js/utilities.js (hunk 2 above)
Comment (tonyurquidi) [reply src/app/js/utilities.js]: removed

File: src/app/js/utilities.js (hunk 1 above)
Comment (tonyurquidi) [reply src/app/js/utilities.js]: removed

**Pull request merged**